airflow/scripts/git_scripts/github_repository.py

```diff
@@ -0,0 +1,99 @@
+from collections import defaultdict
+from datetime import datetime, timedelta
+import os
+from typing import Any, Dict, List
+
+from dotenv import load_dotenv
+
+from github_entity import GitHubEntity
+from repository import Repository
+
+
+load_dotenv()
+github_token = os.getenv('GITHUB_TOKEN')
+
+class FetchRepository(GitHubEntity):
```

Comment:
changed it. but won't that be confusing because the Pedantic class is name Repository and the class that fetches data from the repo is called GitRepository or its fine?